pkg/operator/ceph/cluster/cluster.go

```diff
@@ -52,18 +52,22 @@ var (
 )
 
 type cluster struct {
-	context   *clusterd.Context
-	Namespace string
-	Spec      *cephv1.ClusterSpec
-	mons      *mon.Cluster
-	stopCh    chan struct{}
-	ownerRef  metav1.OwnerReference
+	context              *clusterd.Context
+	Namespace            string
+	Spec                 *cephv1.ClusterSpec
+	mons                 *mon.Cluster
+	stopCh               chan struct{}
+	ownerRef             metav1.OwnerReference
+	orchestrationRunning bool
+	orchestrationPending bool
 }
 
 func newCluster(c *cephv1.CephCluster, context *clusterd.Context) *cluster {
 	return &cluster{Namespace: c.Namespace, Spec: &c.Spec, context: context,
-		stopCh:   make(chan struct{}),
-		ownerRef: ClusterOwnerRef(c.Namespace, string(c.UID))}
+		stopCh:               make(chan struct{}),
+		ownerRef:             ClusterOwnerRef(c.Namespace, string(c.UID)),
+		orchestrationRunning: false,
```

Comment:
false is the default value so it's not necessary to set these bools, but feel free to leave them if you prefer

Reply:
I know. But IMO things like that makes code easier to read even for non-[language-of-your-choice] programmers. ;)